test/demo/findIntentsByContext.inttest.ts

```diff
@@ -1,25 +1,30 @@
 import 'jest';
-import {Context} from '../../src/client/context';
+
 import {ResolveError} from '../../src/client/errors';
+import {Context, AppIntent} from '../../src/client/main';
 
-import {fin} from './utils/fin';
 import * as fdc3Remote from './utils/fdc3RemoteExecution';
+import {testManagerIdentity} from './constants';
+import {setupTeardown} from './utils/common';
 
-const testManagerIdentity = {
-    uuid: 'test-app',
-    name: 'test-app'
-};
-
+/**
+ * A context missing the mandatory `type` field
+ */
 const invalidContext = {
-    twitter: '@testname'
+    twitter: 'testname'
 } as unknown as Context;
 
-describe('Resolving intents by context, findIntentsByContext', () => {
-    beforeEach(async () => {
-        // The main launcher app should remain running for the duration of all tests.
-        await expect(fin.Application.wrapSync(testManagerIdentity).isRunning()).resolves.toBe(true);
-    });
+/**
+ * A context not accepted by any intent in any directory app
+ */
+const unknownContext = {
+    type: 'test.ContextTypeUnknown',
+    name: 'Test Name'
```

Comment:
Perhaps these could be moved into `constants.ts`?
A similar object to `invalidContext` is being used in `open.inttest.ts`.